db/service_updates.yml

```diff
@@ -1,3 +1,30 @@
+  - date: '2023-05-26'
```

Comment:
This date is out of order from the other ones.

Reply:
I thought I'd matched all three, thanks for spotting this. Following our discussion of this, I'm going to change the dates so that each of the news entries has a different date. This will allow us to be certain of the order in which they'll be displayed.